bsip-0085.md

```diff
@@ -0,0 +1,61 @@
+    BSIP: 0085
+    Title: Maker order creation fee discount
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-10-13
+    Discussion: https://github.com/bitshares/bsips/issues/240
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change so that the committee can define
+different fee rates for maker orders and taker orders.
+
+# Motivation
+
+To improve liquidity of the BitShares DEX.
+
+# Rationale
+
+The maker-taker fee model is adopted widely in centralized exchanges and
+helped them to attract trading activities.
+
+In the BitShares DEX, for each filled order, the owner of the order need to
+pay an order creation fee and conditionally a market fee.
+[BSIP 81](bsip-0081.md) described
+a way to apply the maker-taker fee model to the market fee. This BSIP
+describes a way to apply the maker-taker fee model to the order creation fee.
+
+# Specification
+
+Add a new global parameter `maker_fee_discount_percent` which can
+be updated by the committee only after the protocol upgrade.
+Initial value of that parameter is `0`.
+Valid range of that parameter is `[0, 100%]`.
+
+When a limit order is filled or partially filled for the first time,
+technically, when its `deferred_fee` field is non-zero,
+* if the order is a take order, process the deferred fee as before;
+* if the order is a maker order, 
+  * if the order creation fee was paid in BTS, return
+  `round_down(deferred_fee * maker_fee_discount_percent)` to the owner,
+  then process the remaining deferred fee as before;
+  * if the order creation fee was paid in another asset, return
```

Comment:
Can you please elaborate why this has two parts? (deferred paid fee and deferred fee)? @abitmore

Reply:
It's explained in [BSIP 26](https://github.com/bitshares/bsips/blob/master/bsip-0026.md).

Reply:
Thx